x-pack/qa/repository-old-versions/src/test/java/org/elasticsearch/oldrepos/OldMappingsIT.java

```diff
@@ -196,6 +217,12 @@ public void setupIndex() throws IOException {
         assertOK(response);
     }
 
+    private String getRandom(String[] strings) {
+        Random random = new Random();
+        int randomIndex = random.nextInt(strings.length);
+        return strings[randomIndex];
```

Comment:
we should do pseudo random with a seed here. possibly depending on the test seed. Just use one of the methods from LuceneTestCase?

Reply:
mostly you can replace this method with calling EsTestCase#randomFrom